### PR TITLE
OCP4: Add response to SC-13

### PR DESCRIPTION
**Pull request by JAORMX**
Signed-off-by: Juan Antonio Osorio Robles <jaosorior@redhat.com>

### CODE REVIEW

File: openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml [hunk 1]
```diff
@@ -812,10 +812,11 @@
...
   narrative:
     - text: |
-        A complete control response is planned. Engineering progress can be
-        tracked via:
+        At installation time, FIPS mode can be enabled in OpenShift
+        through a flag [1][2]
 
-        https://issues.redhat.com/browse/CMP-420
+        [1] https://docs.openshift.com/container-platform/latest/installing/installing-fips.html
+        [2] https://docs.openshift.com/container-platform/latest/installing/installing_aws/installing-aws-government-region.html#installation-configuration-parameters_installing-aws-government-region
```
Comment (jhrozek) [inline openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml:819]: Nit: why is the AWS doc added explicitly but not azure of GCP?

Comment (JAORMX) [reply openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml:819]: cause I'm lazy. Should I add those links too?

Comment (JAORMX) [reply openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml:822]: done, thanks for the review.

**Review by jhrozek — APPROVED** [review]
Thanks for the changes, this version looks good

**Pull request merged**